parsing/src/recipes.ts

```diff
@@ -39,14 +40,12 @@ function getRecipes(
                     ?.map((ingredientStr: string) => {
                         const match = RegExp(/Desc_(.*?)\.Desc_.*?,Amount=(\d+)/).exec(ingredientStr);
                         if (match) {
-                            const partName = match[1];
-                            let amount = parseInt(match[2], 10);
-
+                            const partName: string = match[1];
+                            let amount: number = parseInt(match[2], 10);
```

Comment:
```suggestion
                            let amount = parseInt(match[2], 10);
```
`parseInt` is typecasted to return a number.